Makefile

```diff
@@ -40,29 +41,66 @@ all: build
 .PHONY: build
 build: build-controller build-scheduler
 
+.PHONY: build.amd64
+build.amd64: build-controller.amd64 build-scheduler.amd64
+
+.PHONY: build.arm64v8
+build.arm64v8: build-controller.arm64v8 build-scheduler.arm64v8
+
 .PHONY: build-controller
 build-controller: autogen
 	$(COMMONENVVAR) $(BUILDENVVAR) go build -ldflags '-w' -o bin/controller cmd/controller/controller.go
 
+.PHONY: build-controller.amd64
+build-controller.amd64: autogen
+	$(COMMONENVVAR) $(BUILDENVVAR) GOARCH=amd64 go build -ldflags '-w' -o bin/controller cmd/controller/controller.go
+
+.PHONY: build-controller.arm64v8
+build-controller.arm64v8: autogen
+	GOOS=linux $(BUILDENVVAR) GOARCH=arm64 go build -ldflags '-w' -o bin/controller cmd/controller/controller.go
+
 .PHONY: build-scheduler
 build-scheduler: autogen
 	$(COMMONENVVAR) $(BUILDENVVAR) go build -ldflags '-X k8s.io/component-base/version.gitVersion=$(VERSION) -w' -o bin/kube-scheduler cmd/scheduler/main.go
 
+.PHONY: build-scheduler.amd64
+build-scheduler.amd64: autogen
+	$(COMMONENVVAR) $(BUILDENVVAR) GOARCH=amd64 go build -ldflags '-X k8s.io/component-base/version.gitVersion=$(VERSION) -w' -o bin/kube-scheduler cmd/scheduler/main.go
+
+.PHONY: build-scheduler.arm64v8
+build-scheduler.arm64v8: autogen
```

Comment:
Ditto. It seems the command is identical with `build-scheduler.amd64`.

Reply:
Fixed in commit 9ca52707ffc1826e7ce53f45c710e00ce0b279ce.